k2/csrc/properties.cc

```diff
@@ -5,39 +5,34 @@
 
 // See ../../LICENSE for clarification regarding multiple authors
 
+#include "k2/csrc/properties.h"
+
 #include <unordered_set>
 
-#include "k2/csrc/properties.h"
 #include "k2/csrc/fsa.h"
 
 namespace k2 {
 
 bool IsTopSorted(const Fsa &fsa) {
```

Comment:
Do we use the style

`const Fsa &fsa` or
`const Fsa& fsa` ?

Reply:
It is from my vim plugin `google/vim-codefmt`, let me check if it is compatible with google style.

Reply:
Have you placed `.clang-format` in your project directory?
I remember that it is checked into the repo.

Reply:
https://github.com/danpovey/k2/blob/57d27c767ebee986b89f348c3cd2e51974e52d86/.clang-format#L7

It has been set to false, so `&` and `*` will always be left aligned.

Reply:
> Do we use the style
> 
> `const Fsa &fsa` or
> `const Fsa& fsa` ?

I've always used `const Fsa &fsa`, but if the other one is officially preferred I'm OK to switch.  Would be easier for me to stay with how Kaldi did it though.

Reply:
@csukuangfj Seems google style did not say about this?

Reply:
OK lets right align them then, like Kaldi

Reply:
Right align is fine for me. Just want to be consistent.